server/index.js

```diff
@@ -5,13 +5,29 @@ const port = parseInt(process.env.PORT, 10) || 3000;
 const dev = process.env.NODE_ENV !== 'production';
 const app = next({ dev });
 const handle = app.getRequestHandler();
+const multer = require('multer');
+const uuidv4 = require('uuid/v4');
 
 const content = require('./content');
 const contentType = require('./content-type');
 
+const createStorage = () => {
+  const currentDate = `${new Date().getMonth() +
```

Comment:
again, `createStorage()` is called only once on bootstrap thus currentDate is still not going to change over the time. You probably want to prepend the date to the filename:
```js
filename(req, file, cb) {
   const currentDate = `${new Date().getMonth() + 1}.${new Date().getDate()}.${new Date().getFullYear()}`;
   cb(null, path.join(currentDate, file.originalname));
  },

```